weblate/templates/about.html

```diff
@@ -2,27 +2,27 @@
 {% load i18n %}
 
 {% block breadcrumbs %}
-<li><a href="{% url 'about' %}">{% trans "About Weblate" %}</a></li>
+<li><a href="{% url 'about' %}">{% trans "About" %}</a></li>
 {% endblock %}
 
 {% block content %}
 
 <div class="container-fluid">
 
 <div class="panel panel-primary">
-<div class="panel-heading"><h4 class="panel-title">{% trans "About Weblate" %}</h4></div>
+<div class="panel-heading"><h4 class="panel-title">{% trans "About" %}</h4></div>
 <div class="panel-body">
 
-<p>{% blocktrans %}Weblate is a web based translation tool with tight version control integration. It features simple and clean user interface, propagation of translations across components, quality checks and automatic linking to source files.{% endblocktrans %}</p>
+<p>{% blocktrans %}Weblate is a web based translation tool with tight version control integration. It features a simple and clean user interface, propagation of translations across components, quality checks and automatic linking to source files.{% endblocktrans %}</p>
 
-<p>{% blocktrans %}If you are looking for support for Weblate or want to file bug reports check its website at <a href="{{ weblate_url }}">weblate.org</a>.{% endblocktrans %}</p>
+<p>{% blocktrans %}If you are looking for support, or want to file bug reports, go to <a href="{{ weblate_url }}">weblate.org</a>.{% endblocktrans %}</p>
```

Comment:
This wording was intentional to differentiate support for Weblate and for given instance. If somebody is running Weblate, we certainly don't want users to ask for instance specific support (eg. about translations or projects available there) in our issue tracker. Still it could be probably written better...